mongodb-atlas-sql-interface.md

```diff
@@ -34,22 +40,35 @@ To connect using the Atlas SQL interface :
 
 3. Select **Database** from the categories on the left, select **MongoDB Atlas SQL**, and then select **Connect**.
 
-2. In the MongoDB Atlas SQL window that appears provide:
-   - The **MongoDB URI** you retrieved previously. Make sure that it does not contain your username and password or it will be rejected.
-   - Your federated **Database** name.
```

Comment:
I believe in Power BI you must enter in the database (even though it is also included in the string)